src/templates/default_nativeapp/app/README.md

```diff
@@ -0,0 +1,22 @@
+## Welcome to your First Native Application!
+
+In this application, you will be able to explore some basic concepts such as application role, versioned schemas and creating procedures and functions within a setup script.
+
+For a deep dive into the world of Native Apps, please read the [official Snowflake documentation](https://docs.snowflake.com/en/developer-guide/native-apps/native-apps-about) which goes in depth about many additional functionalities of this framework.
+
+## Using the application after installation
+To interact with the application after it has successfully installed in your account, switch to the application owner role first.
+
+### Calling a stored procedure
+
+```
+CALL <application_name>.versioned_schema.area_of_square_sproc(2);
```

Comment:
Have you considered templating `<application_name>` so it works out of the box after generating a project?

Reply:
Updated, thanks!